File: Week2/day2/draw-pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

number = int(input('Give me a number, please: '))
for i in range(1,number+1):
   print (" "*(number-i)+"*"*i+"*"*(i-1))

File: Week7/Day2/draw-pyramid.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is
var star = '*';
var space = ' ';
var counter = lineCount;
var pyramid = '';
for (var i = 1; i <= lineCount; i++)
{
    counter -= 1;
    space = ' ';
    for (var h = 1; h <= counter; h++)
    {
        space += ' ';
    }
    pyramid = space + star;
    console.log(pyramid);
    star += "**";
}